canonical-websites/www.ubuntu.com
# Remove /index2.html as no longer used

## Done

* removed the /index2.html page as it is no longer used
* added redirect from index2 to index

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- test that going to [index2](http://0.0.0.0:8001/index2) takes you to the / homepage

## Issue / Card

Fixes #2546
